nexus/tests/integration_tests/commands.rs

```diff
@@ -151,6 +152,7 @@ fn test_nexus_openapi() {
             .or_default()
             .push((
                 op.operation_id.as_ref().unwrap().to_string(),
+                method.to_string(),
```

Comment:
```suggestion
                method.to_string().to_uppercase(),
```

Minor nit here, but I think uppercase versions would be more consistent w/ how method names are generally displayed.